GoogleDataTransport/GDTCORLibrary/GDTCORPlatform.m

```diff
@@ -237,6 +237,14 @@ @interface GDTCORApplication ()
 
 @implementation GDTCORApplication
 
+/** A dispatch queue on which all task semaphore will populate and remove from
+ * sBackgroundIdentifierToSemaphoreMap.
+ */
+static dispatch_queue_t sSemaphoreQueue;
```

Comment:
This is my fault, but rename sSemaphoreQueue and sBackgroundIdentifierToSemaphoreMap to have a g prefix instead of an s.

go/objc-style#common-variable-names

Reply:
fixed